tests/e2e/factories/src/factories/product.ts

```diff
@@ -0,0 +1,22 @@
+import { Factory } from 'fishery';
+import { Product } from '../models';
+import apiFetch from '@wordpress/api-fetch';
+
+export default Factory.define< Product >( ( { params, onCreate } ) => {
+	onCreate( async ( product ) => {
+		return apiFetch( {
+			url: '/wc/v3/products',
+			method: 'POST',
+			data: {
+				type: 'simple',
+				name: product.Name,
+				regular_price: product.RegularPrice,
+			},
+		} ).then( ( data: any ) => {
+			product.onCreated( data.id );
+			return product;
+		} );
+	} );
```

Comment:
As Darren noted in slack, this won't work because Jest is running the code not puppeteer, you don't run code in puppeteer you control the browser via a set of utilities.

I suggest we use https://github.com/node-fetch/node-fetch to keep the same API and spec.

Reply:
I agree, I was looking at that library last week but there was a misunderstanding around the availability of browser globals in the suite. Thank you very much for confirming that this package would be a good choice!